controller/service.go

```diff
@@ -63,31 +69,56 @@ func (s *Service) CreateBirthday(ctx context.Context, req *pb.CreateBirthdayRequ
 	return convertedBirthday, nil
 }
 
+//GetBirthday is getting a birthday object
 func (s *Service) GetBirthday(ctx context.Context, req *pb.GetBirthdayRequest) (*pb.BirthdayObject, error) {
-	birthdayCollection := mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
 
-	birthday := birthdayCollection.FindOne(ctx, bson.M{"personalNumber": req.PersonalNumber})
-	BirthdayModel := &BirthdayStruct{}
+	birthday := s.BirthdayCollection.FindOne(ctx, bson.M{"personalNumber": req.PersonalNumber})
+	BirthdayModel := &BirthdayModel{}
 	birthday.Decode(BirthdayModel)
 	convertedBirthday := BirthdayModel.asBirthdayObject()
 
 	return convertedBirthday, nil
 }
 
+//GetAllBirthday is creating all birthday objects
+func (s *Service) GetAllBirthday(ctx context.Context, req *pb.GetAllBirthdayRequest) (*pb.GetAllBirthdayResponse, error) {
+
+	cursor, err := s.BirthdayCollection.Find(ctx, bson.M{})
+	if err != nil {
+		log.Fatal(err)
+	}
+	var mongoBirthdays []BirthdayModel
+	if err = cursor.All(ctx, &mongoBirthdays); err != nil {
+		log.Fatal(err)
+	}
+	birthdaysArray := make([]*pb.BirthdayObject, 0, len(mongoBirthdays))
+
+	for index := 0; index < len(mongoBirthdays); index++ {
+
+		birthday := mongoBirthdays[index]
+		convertedBirthday := birthday.asBirthdayObject()
+		birthdaysArray = append(birthdaysArray, convertedBirthday)
+
+		fmt.Println(convertedBirthday)
+	}
+
+	return &pb.GetAllBirthdayResponse{Birthdays: birthdaysArray}, nil
+}
+
+// UpdateBirthday is updating a birthday object
 func (s *Service) UpdateBirthday(ctx context.Context, req *pb.UpdateBirthdayRequest) (*pb.BirthdayObject, error) {
-	birthdayCollection := mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
 
 	opts := options.Update().SetUpsert(true)
-	filter := bson.D{{"personalNumber", req.PersonalNumber}}
-	update := bson.D{{"$set", bson.D{{"name", req.Name}, {"date", req.Date}, {"personalNumber", req.PersonalNumber}}}}
-	result, err := birthdayCollection.UpdateOne(context.TODO(), filter, update, opts)
+	filter := bson.D{{Key: "personalNumber", Value: req.PersonalNumber}}
```

Comment:
All of the strings here should be consts as labels. See the constants in permission-service's `store.go`

Reply:
done